stable.rust-toolchain

```diff
@@ -0,0 +1 @@
+stable-2020-10-08
```

Comment:
The naming convention of this file feels a bit strange to me, but maybe there is some convention that I am not aware of?

Reply:
It was the same convention used in the current `rust-toolchain` file.  Instead of the version it's the date of the release.